### ISSUE TITLE
Bad results with custom indoor data using pretrained weights

### ISSUE THREAD

**Issue by houchenfeng** [Haian-Jin/LVSM#18]
Hello, I'm encountering issues with inference results when using the pretrained weights on my own indoor scene dataset. Here's what I've done:
Implemented data loading and interpolated pose generation in dataset_scene.py

Used the provided decoder-only 512x512 resolution pretrained weights for inference
BUT I observed artifacts, missing objects, and incorrect predictions in the results

These issues seem inconsistent with the results presented in the paper, and I'm seeking guidance on how to resolve them. Attached below are my implementation code for my data inference and sample results for reference.
Could you provide insights on potential causes (e.g., pose interpolation method, dataset formatting, hyperparameter tuning) or recommend steps to debug this problem? Thank you!

（here is my code https://github.com/houchenfeng/lvsm-test/blob/ad21ea2d51386a91a7d2581863b70857e3372fd5/data/dataset_scene.py#L415）

https://github.com/user-attachments/assets/3c638f69-97cb-4ffe-b278-f12217b5aecd

**Comment by Haian-Jin** [comment]
Hi!

Did you also use 2 input views here? It's better to also post the input views here.

If the first frame and the last frame of the uploaded videos are the two input images, they seem to have very different focal length and camera pose distributions from the training dataset (realestate10k). If so, it's better fine-tune the model on your dataset.

In addition, please also check and confirm that the input poses have been normalized using the code provided in my dataloader.
